kmp-nativecoroutines-core/src/supportedTargetTest/kotlin/com/rickclephas/kmp/nativecoroutines/NativeFlowTests.kt

```diff
@@ -1,29 +1,31 @@
 package com.rickclephas.kmp.nativecoroutines
```

Comment:
These test classes can be reverted to the state from #44, right? 
I think moving them to the new "common" source set makes sense, we'll deal with the conversion to `coroutines-test` later 🙂

Reply:
Going to rebase now that #44 has been merged 😄